### PR TITLE
New package: nitroshare-0.3.4

### LINKED ISSUE #9033: Package request: Nitroshare
Lan file sharing.
https://nitroshare.net/

### PR DESCRIPTION
**Pull request by Cogitri**
fixes #9033

### CODE REVIEW

**Comment by Cogitri** [comment]
Please test @manipuladordedados, seems to work for me.

**Comment by maxice8** [comment]
Nitroshare has some places that need pycompile_dirs

/usr/share/nautilus-python/extensions/nitroshare.py
/usr/share/nemo-python/extensions/nitroshare.py
/usr/share/caja-python/extensions/nitroshare.py

File: srcpkgs/nitroshare/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'nitroshare'
+pkgname=nitroshare
+version=0.3.4
+revision=1
+wrksrc=nitroshare-desktop-${version}
+build_style=cmake
+hostmakedepends="pkgconf qt5-qmake"
```
Comment (maxice8) [inline srcpkgs/nitroshare/template]: pkg-config

**Comment by Cogitri** [comment]
Fixed.

**Pull request merged**